src/components/TeacherComponents/LessonOverviewHeaderDownloadAllButton/LessonOverviewHeaderDownloadAllButton.tsx

```diff
@@ -32,6 +33,9 @@ export const LessonOverviewHeaderDownloadAllButton: FC<
   } = props;
 
   const preselected = "all";
+  const downloads = useFeatureFlagEnabled("use-auth-owa")
+    ? "downloads-auth"
+    : "downloads";
```

Comment:
I've created a few alternate pages next to the original `downloads` pages. The only difference being that they're wrapped in the `withPageAuthRequired` hoc which will do the check for a logged in user before rendering the page.

This was the only sensible way I could see of enforcing the feature flag while not breaking things for existing users.